crates/layer/src/ckb.rs

```diff
@@ -155,12 +155,7 @@ impl<S: Store<H256> + ClearStore> CkbSimpleAccount<S> {
         let data = BytesOpt::new_builder()
             .set(Some(proof.serialize(program)?.pack()))
             .build();
-        let mut witness_builder = WitnessArgs::new_builder();
-        if self.last_cell.is_none() {
-            witness_builder = witness_builder.output_type(data);
-        } else {
-            witness_builder = witness_builder.input_type(data);
-        }
+        let witness_builder = WitnessArgs::new_builder().output_type(data);
```

Comment:
Can you elaborate on the choice of always sticking to output type here?

Reply:
It's a mistake, I'll revert it.

In polyjuice the situation is more complicated, if the contract cell will be destroyed (only located in input), then put its witness in `input_type` otherwise put in `output_type`